src/cfg/mod.rs

```diff
@@ -26,6 +24,22 @@ mod tests;
 pub(crate) mod structured;
 pub(crate) mod to_structured;
 
+/// Convert a program to a cfg.
+/// Loops over all the functions, translating individually.
+pub(crate) fn program_to_cfg(program: &Program) -> CfgProgram {
```

Comment:
nit: We should decide if we want toplevel functions for converting "xs" to "ys" or if we want these to be methods on the destination type, i.e.

```rust
impl CfgProgram {
fn new(program: &Program) -> CfgProgram { ... }
}
```

I think I've added a few functions like this, but it may be more idiomatic rust to have them as methods. This comment is optional; I'd say it's up to you but we should be consistent.

Reply:
Good question, I'm also not sure. So far I've tried to make all the conversions simply available as methods like `Optimizer::cfg_to_rvsdg`